custom_components/bmw_connected_drive/const.py

```diff
@@ -5,3 +5,8 @@
 CONF_ALLOWED_REGIONS = ["china", "north_america", "rest_of_world"]
 CONF_READ_ONLY = "read_only"
 CONF_USE_LOCATION = "use_location"
+
+CONF_LAST_TRIP = "LAST_TRIP"
```

Comment:
You can remove the `CONF_` part. That is indicating it's a configuration item as part of the config flow.
That is not the case for these constants.